packages/istanbul-reports/lib/html/templates/head.txt

```diff
@@ -56,5 +56,8 @@
       </div>
       {{/if_has_ignores}}
     </div>
+    <p class="quiet">
+      Press <em>n</em> or <em>j</em> to go to next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for previous
```

Comment:
Extremely minor grammatical nitpick, add: `the next uncovered block,` and a period at the end.